tests/generate_packageTest.py

```diff
@@ -136,12 +146,14 @@ def test_register(self, call_mock):
 
     def test_no_bear_object(self):
         if not os.path.exists(self.NO_BEAR_PATH):
-            os.makedirs(os.path.join('bears', 'BadBear'))
+            os.makedirs(os.path.join(self.BEARS_PATH, 'BadBear'))
             touch(self.NO_BEAR_PATH)
         main()
         self.assertFalse(os.path.exists(os.path.join(
-            'bears', 'upload', 'BadBear')))
-        shutil.rmtree(os.path.join('bears', 'BadBear'))
+            self.BEARS_UPLOAD_PATH, 'BadBear')))
+        shutil.rmtree(os.path.join(self.BEARS_PATH, 'BadBear'))
 
     def tearDown(self):
-        shutil.rmtree(os.path.join('bears', 'upload'))
+        shutil.rmtree(os.path.join(self.BEARS_UPLOAD_PATH))
+
+        os.chdir(self.orig_dir)
```

Comment:
im not very sure why we need this, i can't see any "CD"s in the tests, or i may be missing it?